### PR TITLE
FSE: Get plugin ready for directory submission

### PR DESCRIPTION
**Pull request by obenland**
Adds a readme.txt file in order to be able to submit this plugin to the directory.

See #33349.

### CODE REVIEW

File: apps/full-site-editing/full-site-editing-plugin/readme.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+1. Activate the plugin through the 'Plugins' screen in WordPress.
+1. Create a new page and select a template that best suits your needs.
+1. Place the "Blog Posts Listing" block anywhere you want inside the block editor.
+
+
+== Frequently Asked Questions ==
+
+= Can I use this plugin in production? =
+
+We'll be making frequent updates to the plugin as we flesh out its feature set. You're welcome to try it, just be aware that it could break after an update.
```
Comment (apeatling) [inline apps/full-site-editing/full-site-editing-plugin/readme.txt]: "We'll be making frequent updates to the plugin as we flesh out its feature set. You're welcome to try it, just be aware that it is only designed to work on the WordPress.com environment and could break after an update."

File: apps/full-site-editing/full-site-editing-plugin/readme.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+= How is the Blog Posts Listing block different from the Latest Posts block in Core? =
+
+It adds an excerpt! And meta information! It really is much more useful, especially if your looking for a block that gives readers a better idea about your latest posts than just the title.
+
+
```
Comment (apeatling) [inline apps/full-site-editing/full-site-editing-plugin/readme.txt]: = Do you provide support for this plugin? =

This plugin is experimental, so we don't provide any support for it outside of websites hosted on WordPress.com at this time.

File: apps/full-site-editing/full-site-editing-plugin/readme.txt [hunk 3]
```diff
@@ -0,0 +1,41 @@
+=== Full Site Editing ===
+Contributors: automattic, get_dave, marekhrabe, mppfeiffer, obenland
+Tags: block, blocks, editor, gutenberg, page
+Requires at least: 5.0
+Tested up to: 5.2
+Stable tag: 1.0
```
Comment (apeatling) [inline apps/full-site-editing/full-site-editing-plugin/readme.txt]: Can we keep this under 1.0? We're not close to this being a release that is production worthy. Ideally this is a 0.X release.

File: apps/full-site-editing/full-site-editing-plugin/readme.txt [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+== Description ==
+
+This plugin comes with a custom block to display a list of your most recent blog posts, as well as a template selector
+to give you a head start on creating new pages for your site.
+
+
+== Installation ==
+
+1. Upload the plugin files to the `/wp-content/plugins/full-site-editing` directory, or install the plugin through the WordPress plugins screen directly.
+1. Activate the plugin through the 'Plugins' screen in WordPress.
```
Comment (apeatling) [inline apps/full-site-editing/full-site-editing-plugin/readme.txt]: These are all 1. bullet points.

File: apps/full-site-editing/full-site-editing-plugin/readme.txt [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+We'll be making frequent updates to the plugin as we flesh out its feature set. You're welcome to try it, just be aware that it could break after an update.
+
+= How is the Blog Posts Listing block different from the Latest Posts block in Core? =
+
+It adds an excerpt! And meta information! It really is much more useful, especially if your looking for a block that gives readers a better idea about your latest posts than just the title.
+
+
+== Changelog ==
+
+= 1.0 =
```
Comment (apeatling) [inline apps/full-site-editing/full-site-editing-plugin/readme.txt]: Release version should be changed here too.

File: apps/full-site-editing/full-site-editing-plugin/readme.txt (hunk 4 above)
Comment (obenland) [reply apps/full-site-editing/full-site-editing-plugin/readme.txt]: I adopted that from the [readme example](https://wordpress.org/plugins/readme.txt). Markup converts those into an ordered list

**Pull request merged**